ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -358,6 +363,10 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
     setTimeRange({ start: new Date(event.start), end: new Date(event.end) });
   };
 
+  const isStackedBar = visual.display === 'bar' && visual.stack === 'All';
+
+  const tooltipConfig: TooltipConfig = { wrapLabels: true };
```

Comment:
If this object is constant, it might be beneficial to pull the variable definition outside the render function. That way react always sees it as the same value (as opposed to it being recreated on every render cycle).

Reply:
Thanks this is fixed, added DEFAULT_TOOLTIP_CONFIG and can merge in future spec.tooltip properties, going to add one to disable tooltip pinning in a follow up